schema/build-config-example.js

```diff
@@ -36,18 +37,17 @@ fs.writeFileSync(
   "utf8"
 );
 
-const configStringYaml = configStringJson
-  .replace(/JSON\(C\)/, "YAML")
-  .replace(/\n {2}/g, "\n")
-  .replace(/\/\/ /g, "# ")
-  .replace(/(\s*)"([^"]+)":/g, "$1$2:")
-  .replace(/: \{/g, ":")
-  .replace(/\n\}/g, "")
-  .replace(/,\n/g, "\n")
-  .replace(/^\{\n/, "")
-  .replace(/\}$/, "");
+const configStringYaml = yaml.stringify(configExample, {
+  "lineWidth": 0,
```

Comment:
Are all 3 of these customizations necessary to produce the same output as today? If not, I prefer accepting the default vs. being explicit.

Reply:
- `lineWidth` is used for not breaking lines, this makes the comment transformation really hard

- the other two, are mainly to not configure double_quotes for string but leave it as plain for the keys
```
  # Heading style (without the setting)
  style: consistent
```
vs
```
  # Heading style (with the setting)
  style: "consistent"
```

regarding the quotes, this is easy to handle, the other one I would definitely keep.